charts/agent/templates/_deployment-cluster-events-config.tpl

```diff
@@ -5,7 +5,7 @@ extensions:
 
 exporters:
 {{- include "config.exporters.debug" . | nindent 2 }}
-{{ if .Values.observe.entityToken }}
+{{ if .Values.observe.entityToken.create }}
 {{- include "config.exporters.otlphttp.observe.entity" . | nindent 2 }}
 {{ end }}
 {{- include "config.exporters.otlphttp.observe.base" . | nindent 2 }}
```

Comment:
Should we wrap this in a `{{ if .Values.observe.token.create }}`?
I don't think we absolutely need this, so if this doesn't work, I don't mind if we just leave it as is.
But sounds like the conditional block is the right thing to do?

(Same can be said for the other deployment/daemonset's exporters. If they depends on `observe.token`, maybe only enable them when the token is there.